tox.ini

```diff
@@ -0,0 +1,10 @@
+# content of: tox.ini , put in same dir as setup.py
+[tox]
+envlist = py36,py37
+
+[testenv]
+# install pytest in the virtualenv where commands will be executed
+deps = pytest
```

Comment:
lets change this to respect the `requirements-dev.txt`

Reply:
https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints